packages/sdk/contractkit/src/base.ts

```diff
@@ -22,13 +23,15 @@ export enum CeloContract {
   Reserve = 'Reserve',
   SortedOracles = 'SortedOracles',
   StableToken = 'StableToken',
+  StableTokenEUR = 'StableTokenEUR',
   TransferWhitelist = 'TransferWhitelist',
   Validators = 'Validators',
 }
 
-export type StableTokenContract = CeloContract.StableToken
+export type StableTokenContract = CeloContract.StableToken | CeloContract.StableTokenEUR
+export const stableTokenContractArray = [CeloContract.StableToken, CeloContract.StableTokenEUR]
```

Comment:
So for every new stable token contract, we'll need to add to this array and to the types?

Reply:
Yes
The array is only used for some cli commands, but the type is required for how we handle the tokens. I thought that if you are adding one, adding the other should be obvious, haha